apps/site/src/views/Home/components/ApplyButton/ApplyButton.tsx

```diff
@@ -0,0 +1,15 @@
+import Button from "react-bootstrap/Button";
+
+import styles from "./ApplyButton.module.scss";
+
+export default function ApplyButton() {
+	return (
+		<Button
+			className={styles.applyButton}
+			href="https://hack.ics.uci.edu/" // TODO: Change to application site link
+			variant=""
```

Comment:
Nitpick: No need to set `variant` to an empty string

Reply:
React Bootstrap will apply `.btn-primary` without it. I suppose the custom variant should override everything, but to be safe, better to not worry about the primary variant.